pkg/portal/graphql/survey_mutation.go

```diff
@@ -0,0 +1,56 @@
+package graphql
+
+import (
+	"github.com/graphql-go/graphql"
+
+	"github.com/authgear/authgear-server/pkg/portal/model"
+	"github.com/authgear/authgear-server/pkg/portal/session"
+)
+
+var updateSurveyCustomAttributeInput = graphql.NewInputObject(graphql.InputObjectConfig{
+	Name: "UpdateSurveyCustomAttributeInput",
+	Fields: graphql.InputObjectConfigFieldMap{
+		"surveyJSON": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(graphql.String),
+			Description: "Onboarding survey result JSON.",
+		},
+	},
+})
+
+var _ = registerMutationField(
+	"updateSurveyCustomAttribute",
```

Comment:
Lets name the mutation `saveOnboardingSurvey`. Please rename related type names as well.